incubator/external-dns/templates/deployment.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels: {{ include "labels" . | indent 4 }}
+spec:
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels: {{ include "labels" . | indent 8 }}
+    spec:
+      containers:
```

Comment:
should include the ability to apply resource restrictions. https://github.com/kubernetes/charts/blob/master/stable/uchiwa/templates/deployment.yaml#L21